ui/src/app/utils/errors.tsx

```diff
@@ -85,3 +90,134 @@ export async function fetchWithSystemErrorHandler<T>(
     }
   }
 }
+
+// TODO handle errorClassName and parameters?
+export const defaultErrorResponseFormatter = (
+  errorResponse: ErrorResponse
+): string => {
+  const { message, statusCode, errorCode, errorUniqueId } = errorResponse;
+
+  const errorCodeStr =
+    errorCode && errorCode !== ErrorCode.PARSEERROR
+      ? ` of type ${errorCode.toString()}`
+      : '';
+  const messageStr = message ? `: ${message}` : '.';
```

Comment:
Encode message to remove HTML escaping?

Reply:
Do you have an example of where this would be necessary?

Reply:
if I create a corhort/dataset/something else with  ‘<’, ‘>’, ‘alert’. inside. 
Would that lead to XSS attack if we want to prin ```balbla not found```